terraform/vultr/scripts/deploy.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+
+apt install -y vnstat
+
+docker run -d -it --rm --name=d1000bn --pull always ghcr.io/arriven/db1000n-advanced
```

Comment:
Typo in the name and can you use regular image with `-e ENABLE_PRIMITIVE=false`? The effect would be the same but I want to get rid of advanced image eventually as it takes a bit too long to be built